### PR TITLE
Fix workmode 3 backlight setting

### PR DESCRIPTION
**Pull request by BartMassey**
The code for setting the backlight with xbacklight (workmode 3) was incomplete. This patch finishes it.

### CODE REVIEW

File: xbacklight.c [hunk 1]
```diff
@@ -17,9 +18,9 @@ int get_screen_xbacklight_value() {
 
 int set_screen_xbacklight_value(int backlight) {
 
-	char name[50];
+	char name[256];
 
-	printf("%s\n",name);
+	sprintf(name, "xbacklight -set %.3d\n", backlight);
```
Comment (gaul) [inline xbacklight.c]: snprintf

File: xbacklight.c [hunk 2]
```diff
@@ -28,9 +29,12 @@ int set_screen_xbacklight_value(int backlight) {
 	}
 	const int BUFSIZE = 1000;
 	char buf[ BUFSIZE ];
+        errno = 0;
```
Comment (gaul) [inline xbacklight.c]: strange indentation

File: Makefile [hunk 3]
```diff
@@ -1,8 +1,9 @@
 DESTDIR?=/
 SHELL = /bin/sh
-CC?=gcc
-CFLAGS = `pkg-config --cflags dbus-1 --cflags dbus-glib-1` -Wall -Wextra -Wwrite-strings -O -g
-LDFLAGS= `pkg-config --libs dbus-1 --libs dbus-glib-1` -lX11 -lXext -lXss -lm
+CC=gcc
+CDEBUGFLAGS= -g -O2
+CFLAGS = `pkg-config --cflags dbus-1 --cflags dbus-glib-1` -Wall -Wextra -Wwrite-strings $(CDEBUGFLAGS)
+LDFLAGS= `pkg-config --libs dbus-1 --libs dbus-glib-1` $(CDEBUGFLAGS) -lX11 -lXext -lXss -lm
```
Comment (gaul) [inline Makefile:6]: intentional?

Comment (BartMassey) [inline Makefile:6]: I dunno. Does make it easier to debug, though. Probably should have left the question mark in, I guess.

File: xbacklight.c (hunk 2 above)
Comment (BartMassey) [inline xbacklight.c]: My editor was set not to use tab characters. Fixed.

File: xbacklight.c (hunk 1 above)
Comment (BartMassey) [inline xbacklight.c]: I think %.3d is good enough here, but sure, if you want to. Fixed.

**Comment by BartMassey** [comment]
Thanks! --Bart

**Pull request merged**